packages/hooks/README.md

```diff
@@ -0,0 +1,105 @@
+# Slack CLI Hooks
+
+The `@slack/hooks` package contains scripts that implement the contract between
```

Comment:
Should we name this directory/project `cli-hooks` to make it clear what it's used for, or is there a potential alternative use case that we're omitting that for?